### PR TITLE
AtlasDB Integration

### PR DESCRIPTION
**Pull request by bstadt**
Implements AtlasDB vectorstore for integrating with Atlas by Nomic.
Read more about Atlas at docs.nomic.ai

### CODE REVIEW

**Comment by zanussbaum** [comment]
really would be awesome to have this integration!

**Comment by avysogorets** [comment]
Can't wait to see this!

**Review by hwchase17 — COMMENTED** [review]
this is great. might be nice to have some documentation and example notebook (see other vectorstores) to help people use

**Comment by bstadt** [comment]
> this is great. might be nice to have some documentation and example notebook (see other vectorstores) to help people use

Agreed - adding notebook shortly.

**Comment by theodormarcu** [comment]
This is awesome! Excited for this integration.

**Comment by bstadt** [comment]
notebook integration also embeds the map now ;)
image

File: tests/integration_tests/vectorstores/test_atlas.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+"""Test Atlas functionality."""
+import time
+
+from langchain.docstore.document import Document
+from langchain.vectorstores import AtlasDB
+from tests.integration_tests.vectorstores.fake_embeddings import FakeEmbeddings
+
+ATLAS_TEST_API_KEY = "7xDPkYXSYDc1_ErdTPIcoAR9RNd8YDlkS3nVNXcVoIMZ6"
```
Comment (hwchase17) [inline tests/integration_tests/vectorstores/test_atlas.py:8]: did you mean to check this in?

**Pull request merged**